packages/scm/src/browser/scm-service.ts

```diff
@@ -107,14 +105,4 @@ export class ScmService {
         return repository;
     }
 
-    protected updateContextKeys(): void {
```

Comment:
Did you test VS Code extensions? I.e. that commands are available in the command palette properly. The PR seems to change the semantic when we assign it. We need to test that it still confirms to VS Code extensions' expectations.

Reply:
It seems to work.  I had to hack the builtin I built myself to check the new menu location but it would be good to see the folder menu working when the 1.39 builtin works.  I left the 'string' URI though I don't know if it is used.